Chameleon.Core/Pages/HomePage.xaml

```diff
@@ -104,13 +104,23 @@
                                     iOS="False"
                                     Android="True" />
                             </pan:PancakeView.HasShadow>
-                            <ImageButton
-                                HorizontalOptions="FillAndExpand"
-                                Padding="50"
-                                VerticalOptions="FillAndExpand"
-                                BackgroundColor="#FF252525"
-                                Command="{Binding Source={x:Reference homePage}, Path=BindingContext.AddPlaylistCommand}"
-                                Source="icon_plus" />
+                            <AbsoluteLayout>
+                                <ImageButton
+                                    AbsoluteLayout.LayoutFlags="All"
+                                    AbsoluteLayout.LayoutBounds=".5,.5,1,1"
+                                    Padding="50"
+                                    BackgroundColor="#FF252525"
+                                    Command="{Binding Source={x:Reference Name=homePage}, Path=BindingContext.DataContext.AddPlaylistCommand}"
+                                    Source="icon_plus" />
+                                <Label
+                                    Text="{Binding Source={x:Reference Name=homePage}, Path=BindingContext.DataContext.AddPlaylistLabel}"
```

Comment:
MvxLang werkt hier niet?

Reply:
Dat is de reden dat ik het zo heb gedaan